src/media/local-audio-track.ts

```diff
@@ -0,0 +1,89 @@
+import { logger } from '../util/logger';
+import { LocalTrack } from './local-track';
+
+export type AudioDeviceConstraints = {
+  deviceId?: string;
+};
+
+export type StaticAudioEncoderConfig = {
+  low: '16kHz';
```

Comment:
we need more than just "low" and this should be done in the same way like staticVideoEncoderConfig - with an object containing populated instances of AudioEncoderConfig, so the type would be:
```
export type StaticAudioEncoderConfig: {[key: string]: AudioEncoderConfig}
```

Reply:
Not sure how i missed this but @hmcgowan  is working on it